e2e/features/steps/feedback.py

```
@@ -0,0 +1,120 @@
+    access_feedback_put_endpoint(context, payload)
+
+
+def access_feedback_put_endpoint(context: Context, payload: dict) -> None:
+    """Update feedback using a JSON payload."""
+    assert context is not None
+    endpoint = "feedback/status"
+    base = f"http://{context.hostname}:{context.port}"
+    path = f"{context.api_prefix}/{endpoint}".replace("//", "/")
+    url = base + path
+    headers = context.auth_headers if hasattr(context, "auth_headers") else {}
+    response = requests.put(url, headers=headers, json=payload)
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)